boards/sky-drones/smartap-airlink/init/rc.board_sensors

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# SMARTAP AIRLINK specific board sensors init
+#------------------------------------------------------------------------------
+board_adc start
+
+# Internal SPI bus MPU-9250 / SPI 2 [FPC]
+mpu9250 -s -b 2 -M -R 6 start
```

Comment:
```suggestion
mpu9250 -s -b 2 -R 6 start
```

I would skip the mpu9250 mag if you have another internal magnetometer working. Long story, but when the mpu9250 ak8963 is enabled is disrupts the FIFO output and you don't get clean 8 kHz raw data.

Reply:
If you plot `vehicle_imu_status.gyro_raw_rate_hz` or look at `sensors status` you'll be able to see the difference.

Reply:
Thanks! Then it makes sense indeed. I would still prefer to have magnetometer enabled on one mpu9250 but not on both for redundancy in addition to onboard bmm150.